app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
+    total = 0
+
+    for i in range(len(friends)):
+        try:
+            cafe.visit_cafe(friends[i])
+        except (NotVaccinatedError, OutdatedVaccineError):
+            return 'All friends should be vaccinated'
+        except NotWearingMaskError:
+            total += 1
+
+    if total:
+        return f'Friends should buy {total} masks'
+    else:
```

Comment:
Do not use else condition because if the previous condition is not True you will return this expression anyway, just write return `f"Friends can go to {cafe.name}" and shift it to the left by 4 spaces`.